tools/monitoring/check-reserved.ts

```diff
@@ -0,0 +1,309 @@
+import { typesBundle } from "moonbeam-types-bundle";
+import { ApiPromise, WsProvider } from "@polkadot/api";
+import { BlockHash, PreimageStatus } from "@polkadot/types/interfaces";
+import { xxhashAsU8a } from "@polkadot/util-crypto";
+import { u8aConcat, u8aToHex } from "@polkadot/util";
+import yargs from "yargs";
+import { NETWORK_YARGS_OPTIONS } from "../utils/networks";
+import { promiseConcurrent } from "../utils/functions";
+
+// TODO: clean the code, use PolkadotJS for storage key and ...
```

Comment:
So you mean use `api..query.parachainStaking.nominatorState(acc)` instead of manually building storage keys? Because I was wondering why it was necessary....